auto_gptq/nn_modules/qlinear.py

```diff
@@ -68,6 +68,8 @@ def __init__(
                                         [0, 2, 5, 8, 11, 14, 17, 20, 23, 26, 29, 0],
                                    ], 
                                    dtype=torch.int32).reshape(1, 3, 12)
+        if infeatures % 256 != 0 or outfeatures % 256 != 0:
+            _quant_cuda_available = False
```

Comment:
`_quant_cuda_available` is a global variable, change its value may effect other models initialized after this to use `quant_cuda` kernel, its better to change `self.quant_cuda_available`'s value after line 75